stdlib/Cargo.toml

```diff
@@ -32,6 +32,7 @@ with-debug-info = []
 
 [dependencies]
 assembly = { package = "miden-assembly", path = "../assembly", version = "0.11", default-features = false }
+vm-core = { package = "miden-core", path = "../core", version = "0.11", default-features = false }
```

Comment:
Why do we need to add this dependency?

Reply:
because of this line:
```rust
::vm_core::assert_matches!
```

Reply:
Shouldn't it be a dev dependency then?

Reply:
Updated in 6789011053d76fa41b71046aff8c6d16270a9973.